### PR TITLE
Add a disk status diagnostic based on a touch file in /tmp

### PR DESCRIPTION
**Pull request by Vagabond**

### CODE REVIEW

File: src/gateway_config.erl [hunk 1]
```diff
@@ -228,10 +228,16 @@ diagnostics(Proxy) ->
                         lager:notice("Failed to get p2p status: ~p", [Error]),
                         []
                 end,
+    DiskStatus = case filelib:is_file("/tmp/disk-failure") of
```
Comment (madninja) [inline src/gateway_config.erl:231]: Could you just move this into a separate function and add it to the Base list above?

**Review by madninja — CHANGES_REQUESTED** [review]

**Comment by cokes518** [comment]
Any updates on this PR?

**Comment by JayKickliter** [comment]
Depends on https://github.com/helium/nextgate/pull/765

**Comment by JayKickliter** [comment]
@cokes518 this is ready to merge. We need to make sure the extra field doesn't break the app, at a minimum. And ideally the app should report SD card failure.

**Review by JayKickliter — APPROVED** [review]

**Pull request merged**